app/src/main/java/com/zeapo/pwdstore/PasswordEntry.kt

```diff
@@ -162,7 +164,16 @@ class PasswordEntry(private val content: String) {
     }
 
     companion object {
-
-        private val USERNAME_FIELDS = arrayOf("login:", "username:")
+        @VisibleForTesting(otherwise = PRIVATE)
+        val USERNAME_FIELDS = arrayOf(
```

Comment:
I use user, which does not seem to be included here.

Reply:
It wasn't in the list you proposed in the issue, either. I'll push another patch in a bit.